RecoBTag/ONNXRuntime/plugins/ParticleNetSonicJetTagsProducer.cc

```diff
@@ -188,21 +189,23 @@ void ParticleNetSonicJetTagsProducer::acquire(edm::Event const &iEvent, edm::Eve
           if (i == 0 && (!input_shapes_.empty())) {
             input_shapes_[igroup][2] = insize;
           }
-
           if (debug_) {
-            LogDebug("acquire") << "<ParticleNetSonicJetTagsProducer::produce>:" << std::endl
-                                << " -- var=" << varname << ", center=" << info.center << ", scale=" << info.norm_factor
-                                << ", replace=" << info.replace_inf_value << ", pad=" << info.pad << std::endl;
+            std::cout << " -- var=" << varname << ", center=" << info.center << ", scale=" << info.norm_factor
```

Comment:
please restore all LogDebugs, as these will be required to be allowed in official-cmssw.

Reply:
Fixed